spk/ps3netsrv/src/service-setup.sh

```diff
@@ -1,13 +1,9 @@
 CFG_FILE="${SYNOPKG_PKGDEST}/var/ps3netsrv.conf"
-BIN="${SYNOPKG_PKGDEST}/bin/ps3netsrv"
+BIN_FILE="${SYNOPKG_PKGDEST}/bin/ps3netsrv"
 
-if [ -r "${CFG_FILE}" ]; then
-    . "${CFG_FILE}"
-fi
+export PID_FILE LOG_FILE CFG_FILE BIN_FILE
 
-SERVICE_COMMAND="/bin/stdbuf -o L -e L ${BIN} ${PS3_DIR}"
-SVC_BACKGROUND=y
-SVC_WRITE_PID=y
+SERVICE_COMMAND="${SYNOPKG_PKGDEST}/ps3netsrv-starter.sh"
```

Comment:
I propose to move `SERVICE_COMMAND` to `Makefile` as other related options `SVC_` vanished.

Reply:
Done